blueprints/ember-electron/files/testem-electron.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-env node */
 module.exports = {
-  test_page: 'tests/index.html?hidepassed',
+  test_page: 'tests/index.html?hidepassed=1',
```

Comment:
Forgive my ignorance, but what does the `=1` do?

Reply:
Oh, I should revisit that or add a comment. It doesn't actually make a difference in-browser, but some of the URL munging that we do in `test-runner.js` was resulting the the query string looking like `?hidepassed=&testemid=whatever` and apparently that was preventing QUnit from recognizing the `hidepassed` query param. Giving at value fixed it.

Reply:
I fixed it in `test-runner.js` and removed the `=1`